app/views/index.scala.html

```diff
@@ -6,6 +6,9 @@
     <div class="mdl-grid">
         <div class="mdl-cell mdl-cell--12-col">
             <h1>Choose files to upload</h1>
+            <div class="upload-error hidden" id="png-error">
```

Comment:
I'm using the css libraries 'hidden' class. 

i guess i'll make it upload-error upload-error--hidden ?